js/siteorigin-panels/dialog/widget.js

```diff
@@ -156,12 +156,12 @@ module.exports = panels.view.dialog.extend( {
         if( !this.model.get('missing') ) {
             // Only get the values for non missing widgets.
             var values = this.getFormValues();
-            if (typeof values.widgets === 'undefined') {
+            if ( typeof values.widgets === 'undefined' ) {
```

Comment:
I've been pretty bad about style guidelines, but I think we should follow the WordPress JS conventions. They're pretty similar to the PHP coding guidelines, so easier to keep in mind.

At some point, I'd like to do a complete run through all our code to fix style. Even running it through an PHPStorm code formatting would help.